lua/gk.lua

```diff
@@ -42,8 +42,11 @@ return function (net_conf, lls_conf, sol_conf, gk_lcores)
 	local back_icmp_msgs_per_sec = 1000
 	local back_icmp_msgs_burst = 50
 
+	local co_num = 4
+
 	-- These variables are unlikely to need to be changed.
 	local bpf_enable_jit = true
+	local co_stack_size_kb = 8
```

Comment:
As this parameter may affect the correctness of Gatekeeper, we may have a comment on it, so that others who need to change this value should be aware of this.